src/hotspot/share/gc/shared/gcConfiguration.cpp

```diff
@@ -66,18 +71,22 @@ GCName GCConfiguration::old_collector() const {
     return ParallelOld;
   }
 
+  if (UseShenandoahGC) {
+#if INCLUDE_SHENANDOAHGC
+    if (strcmp(ShenandoahGCMode, "generational") == 0) {
+      return Shenandoah;
+    }
+#endif
+    return NA;
```

Comment:
This looks like a changed behaviour: when generational mode is not selected, we now return NA, while before we returned Shenandoah. We should probably keep returning Shenandoah there.

Reply:
Agree. Will fix.